Lib/test/test_socket.py

```diff
@@ -5276,8 +5276,14 @@ def bind(self, sock, path):
                 raise
 
     def testUnbound(self):
-        # Issue #30205 (note getsockname() can return None on OS X)
-        self.assertIn(self.sock.getsockname(), ('', None))
+        if sys.platform == 'win32':
+            # Getting the name of unbound socket on Windows
+            # raises an exception
```

Comment:
Does this make more sense than returning `None`? Which is apparently a valid return value